src/main/java/lotto/view/ResultView.java

```diff
@@ -0,0 +1,55 @@
+package lotto.view;
+
+import lotto.model.LottoNumber;
+import lotto.model.LottoNumbers;
+import lotto.model.LottoPrizeMap;
+import lotto.model.LottoPrizeRank;
+import lotto.model.LottoPrizeRanks;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static lotto.model.LottoPrizeRank.*;
+
+public class ResultView {
+    private static final String QUANTITY_MESSAGE = "%d개를 구매했습니다.\n";
+    private static final String STATICS_TITLE = "당첨통계\n---------";
+    private static final String STATICS_BODY = "%d개 일치 (%d원)- %d개\n";
+    private static final String RATIO_MESSAGE = "총 수익률은 %.2f입니다\n";
+
+    private static final List<LottoPrizeRank> table = Arrays.asList(THREE, FOUR, FIVE, SIX);
+
+    public static void quantity(int quantity){
```

Comment:
```suggestion
    public static void quantity(int quantity) {
```

전체적으로 이런 부분에 대한 코딩 컨벤션이 지켜지지 않는 부분이 많은 것 같은데요~
코드 작성하고 마지막에 자동으로 코드 프리티 해주는 기능을 혹시 사용하실까요??

참고: https://gocoder.tistory.com/2440

Reply:
앗! 아뇨!! 이 부분도 종종 듣게되는데 습관이 안잡혔네요ㅜㅜ 이번기회에 꼭 잡아봐야겠어요!! 감사드립니다 😀